chrome/WhiteSur/parts/csd.css

```diff
@@ -54,6 +54,20 @@ GNOME CSD styles for headerbar on Firefox [tabsintitlebar] @namespace xul url("h
 	top:10px !important;
 }
 
+/* Fix the botton order */
```

Comment:
Could this be moved to the /parts/windows.css specific CSS not to conflict with other users?

Reply:
As I look into it, many of the CSS that I added are already existent in `windows.css`. I think maybe `@media screen and (-moz-windows-compositor)` doesn’t work any more. Deleting the media tag might fix the problem and this pull request is no longer needed.

Reply:
Thanks for picking that up. I've updated from the media query to the new query targeting windows via `@media  (-moz-platform: windows)` instead. Merging with master